node-drainer/pkg/evaluator/evaluator.go

```diff
@@ -268,14 +270,24 @@ func isNodeAlreadyDrained(ctx context.Context, database queue.DataStore, nodeNam
 		return false, fmt.Errorf("failed to decode result for node %s: %w", nodeName, err)
 	}
 
+	// Try lowercase first (MongoDB compatibility)
 	healthEventStatus, ok := document["healtheventstatus"].(map[string]interface{})
 	if !ok {
-		return false, fmt.Errorf("invalid healtheventstatus format for node %s", nodeName)
+		// Try camelCase (PostgreSQL JSON)
+		healthEventStatus, ok = document["healthEventStatus"].(map[string]interface{})
```

Comment:
Would it be possible to handle this in the `result.Decode` so that we don't have to make this check in every call site?